setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
```

Comment:
`setup/paths.sh` in `vfatqc-python-scripts` is checking that `gem-plotting-tools` has been setup correctly since `trimChamber.py` requires `gem-plotting-tools/macros/fitScanData.py` (which will be moved to `gem-plotting-tools/fitting/fitScanData.py` as requested below).